var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -159,6 +161,7 @@ class Sirius(CMakePackage, CudaPackage, ROCmPackage):
     variant(
         "profiler", default=True, description="Use internal profiler to measure execution time"
     )
+    variant("libsci", default=False, description="Use cray-libsci")
```

Comment:
can't you do a "when libsci is the provider for lapack/scalapack, then..." instead of this explicit option?